### ISSUE TITLE
SEGSAdmin Account Creation allows for username to exceed the amount of characters the client allows

### ISSUE THREAD

**Issue by JamesDW1** [Segs/Segs#437]
### Steps to reproduce behavior/symptoms
1. Add New User in SEGSAdmin with a username of 15+ characters
2. When logging into client, you will be limited to 14 characters

Note: You don't need to run AuthServer to observe that the client limits the username to 14 characters.
### SEGS version
Latest master
### Operating system
Windows 10

**Comment by JamesDW1** [comment]
Just going to quickly add that the same exact circumstances apply to the passwords as well.

**Comment by broxen** [comment]
Good catch! We'll want to ensure that all fields that take input are capped at their max field length if applicable. Username, pass, Map Folder name, etc.

**Comment by miles200** [comment]
@broxen Where can I find max values for fields? Is it easily findable within the code or is it already documented somewhere?

**Comment by broxen** [comment]
This is a great question. Unfortunately I do not believe they are documented in the code. I would check online resources for anticipated values.

For things like Max Slots, the client can't visually (UI) handle more than 8 at this time. Later clients would paginate, but ours does not.

For fade-in, I would set the max around 1000.0, but maybe we should provide some helper text with a suggested range? 260-500 or something.

Since server name isn't actually used, I'm not sure it needs one, but for many of these short string values we should probably assume 32 chars.

**Comment by miles200** [comment]
@broxen This is nearly finished locally, I added a screenshot with some notes in Discord for you to check, just adding the below so I remember as per discussion with @nemerle 

```
hmm.. we'll need to check, but I think I1 client does not support  UTF8 chars in login/pass, so we'll need to at least check login.toLatin1()==login ?
```
